src/main/resources/META-INF/resources/primefaces/picklist/picklist.js

```diff
@@ -651,9 +651,10 @@ PrimeFaces.widget.PickList = PrimeFaces.widget.BaseWidget.extend({
     },
 
     transfer: function(items, from, to, type) {
-        var $this = this,
-        itemsCount = items.length,
-        transferCount = 0;
+        $(this.jqId + ' ul').sortable('disable');
+        var $this = this;
```

Comment:
Should you store $(this.jqId + ' ul') in a variable since you use it twice in this method rather than calculate it 2 times for the enable and disable?

Reply:
Well, two `$(this.jqId + ' ul')` are not in the same function. enable is located in the fireTransferEvent since  it is a callback, but we have to start the disable right at the beginning of the transfer function.

Reply:
Ahh yep I didn't see the code fold I thought it was the same method!

Reply:
Here is the sortable definition

https://github.com/primefaces/primefaces/blob/cf20b82e386a5d483dd17476a62f04832d3f0768/src/main/resources/META-INF/resources/primefaces/picklist/picklist.js#L52-L54